thijsvanloef/palworld-server-docker
# Change Auto Update to use ManifestID and not BuildID

## Context 

* I realized that if there is a new update on Steam for the Palworld dedicated server, the BuildID does not change on every update. To make this realiable we should use the manifestID to check for update.
* This ID can be found in the same files which are currently used.

## Choices

* The API call is the same and it now gets the manifestID instead of the buildID from the response.

## Test instructions

1. Start the server and wait for it to be up
2. Run the command `docker exec palworld-server update` and verify that the server is up to date
3. Change the "manifest" value in `palworld\steamapps\appmanifest_2394010.acf` on Line 31 to a different value
4. Run the command `docker exec palworld-server update` and verify that an update is found

## Checklist before requesting a review

* [ ] I've added documentation about this change to the README. (I don't think the docs need to be updated for this)
